host-controller/src/main/java/org/jboss/as/domain/controller/operations/coordination/ServerOperationResolver.java

```diff
@@ -287,6 +288,15 @@ private static ModelNode cleanOpForDontPropagate(ModelNode op) {
         }
     }
 
+    private static ModelNode getDomainModel(OperationContext context) {
+        ModelNode model = context.getAttachment(DOMAIN_MODEL_ATTACHMENT);
+        if (model == null) {
+            model = Resource.Tools.readModel(context.readResourceFromRoot(PathAddress.EMPTY_ADDRESS, true));
+            context.attach(DOMAIN_MODEL_ATTACHMENT, model);
```

Comment:
Yeah, it's only used here.

It's a clone of the original model, so the only risk is this class modifying it and messing itself up.